be/src/formats/orc/orc_chunk_reader.cpp

```diff
@@ -450,6 +451,7 @@ Status OrcChunkReader::read_next(orc::RowReader::ReadPosition* pos) {
             return Status::EndOfFile("");
         }
     } catch (std::exception& e) {
+        LOG(WARNING) << boost::stacktrace::stacktrace();
```

Comment:
Could you use get_stack_trace in be/src/util/stack_util.cpp?

Reply:
changed